Wheres-Waldo-Backend/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "wheres-waldo-backend",
+	"version": "0.0.0",
+	"private": true,
+	"scripts": {
+		"start": "nodemon ./bin/www"
```

Comment:
Might want a build script in here too, if there's a build step. I'd add a basic README too